### PR TITLE
add admin configuration of file formats

### PR DESCRIPTION
**Pull request by leonidastri**
As a functional admin I want to configure lists of preservable file formats  so that I ensure publication terms are up to date.

**Features added:**

As a functional admin the admin can navigate to Administration page and:

- they can now upload a new file of file formats.
- they can check the current list of files of file formats.
- they can choose a file of file formats and delete it. Then, the file is moved to trash folder, and there is a possibility to retrieve it in case of wrong deletion.

### CODE REVIEW

**Comment by leonidastri** [comment]
This should be reviewed in combination with the following pr in yoda:
 https://github.com/UtrechtUniversity/yoda/pull/477

File: admin/templates/admin/admin.html [hunk 1]
```diff
@@ -80,6 +81,33 @@ <h2 class="card-title">Publication Terms</h2>
...
+                                <select class="form-control" id="file-formats-list" name="filename" required>
+                                    <option value="" disabled selected>Select File Formats to delete</option>
+                                </select>
+                            </div>
+                            <button type="submit" class="btn btn-danger" id="delete-format-button" disabled>Delete File Formats</button>
+                        </form>
+                    </div>
+                    <div>
+                        <div class="mb-3">
+                            <label>Upload new File Formats:</label>
```
Comment (lwesterhof) [inline admin/templates/admin/admin.html]: Singular: `Upload new file format list`

File: admin/templates/admin/admin.html [hunk 2]
```diff
@@ -80,6 +81,33 @@ <h2 class="card-title">Publication Terms</h2>
...
+                    <div class="mb-3">
+                        <label>List of current File Formats:</label>
+                        <form action="{{ url_for('admin_bp.delete_file_formats') }}" method="post" class="flex-fill me-2 needs-validation" novalidate>
+                            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+                            <div class="mb-3">
+                                <select class="form-control" id="file-formats-list" name="filename" required>
+                                    <option value="" disabled selected>Select File Formats to delete</option>
+                                </select>
+                            </div>
+                            <button type="submit" class="btn btn-danger" id="delete-format-button" disabled>Delete File Formats</button>
```
Comment (lwesterhof) [inline admin/templates/admin/admin.html]: Singular: `Delete file format list`

File: admin/templates/admin/admin.html [hunk 3]
```diff
@@ -80,6 +81,33 @@ <h2 class="card-title">Publication Terms</h2>
                         </form>
                     </div>
                 </div>
+
+                <!-- File Formats Section -->
+                <div class="mb-4">
+                    <h2 class="card-title">File Formats</h2>
```
Comment (lwesterhof) [inline admin/templates/admin/admin.html]: Rename to `Preservable file formats` and add a short description. And maybe a link to the file format description in the technical documentation.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

**Comment by leonidastri** [comment]
Updated the code according to the comments!

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**